easybuild/easyconfigs/p/Python-bare/Python-bare-3.6.2.eb

```diff
@@ -20,7 +20,6 @@ sources = ['Python-%(version)s.tgz']
 dependencies = [
```

Comment:
Since you're changing the dependencies here, you could also change the toolchain version to:
```
toolchain = {'name': 'dummy', 'version': ''}
```
With the `dummy` toolchain version I think EB does not actually load the dependencies when building (not sure if this is a bug or a feature).

BTW, in the future we should consider stop using dummy and use a proper toolchain for building python-bare (`GCC`) for example.

Reply:
Sure, I'll do that.